lib/url.c

```diff
@@ -2127,11 +2118,13 @@ static CURLcode parseurlandfillconn(struct Curl_easy *data,
        * However in RFC 1738 the "/" between the host (or port) and the
        * URL-path was NOT part of the URL-path.  Any agent that followed the
        * older spec strictly, and wanted to refer to a file with an absolute
-       * path, would have included a second slash.  So if there are two
-       * slashes, swallow one.
+       * path, would have included a second slash.  POSIX requires the OS to
+       * handle leading double slashes (or triple or quadruple) so there is
```

Comment:
curl builds on numerous not-quite-POSIX platforms as well so I don't think the phrasing should this "absolute" here.

Reply:
Is changing it from "there is no need" to "there should be no need" enough?

Reply:
I could just remove this entire giant comment block since there is no longer any actual code here implementing something. I guess the only reason I left it here was to include the comment about how stripping the extra slash would actually break Windows UNC paths.